cypress/e2e/register.feature

```diff
@@ -0,0 +1,32 @@
+Feature: The Conduit registration page
+
+    Scenario Outline: TC5 - User registers with already taken username
+        Given User is located on the 'register' page of Conduit website
+        When User registers with username 'asd', email '<random_email>' and password 'asd'
+        Then User should see 'username has already been taken' error message
+
+    Scenario Outline: TC6 -  User registers with already taken email
+        Given User is located on the 'register' page of Conduit website
+        When User registers with username '<random_username>', email 'asd' and password 'sdas'
```

Comment:
@karenn32 The documentation says: <random_username>, hovewer, it is no a value but a parameter fo the step in the code.